docs/src/__examples__/Sticky/DEFAULT.tsx

```diff
@@ -3,15 +3,22 @@ import { Sticky, Text } from "@kiwicom/orbit-components";
 
 export default {
   Example: () => (
-    <div style={{ height: "240px" }}>
+    <div style={{ height: "400px", overflow: "scroll" }}>
       <Sticky>
         <Text>Scroll down and see this text stick to the top of the screen.</Text>
       </Sticky>
     </div>
   ),
-  info: {
-    title: "Default sticky",
-    description:
-      "By default, sticky components elevate their children and keep them at the top of the screen.",
-  },
+  exampleKnobs: [
+    {
+      component: "Sticky",
+      knobs: [
+        {
+          name: "offset",
+          type: "number",
+          defaultValue: 10,
+        },
+      ],
+    },
+  ],
```

Comment:
This example doesn't work because sandbox takes the height of what was supposed to be the scrollable area 😅 😅